pkgs/development/python-modules/pygatt/default.nix

```diff
@@ -29,9 +29,12 @@ buildPythonPackage rec {
     pexpect
   ];
 
+  nativeBuildInputs = [
+    nose
+  ];
+
   checkInputs = [
     mock
-    nose
```

Comment:
`nose` is a test requirement. The proper fix would be to patch upstream's `setup.py` file.

Reply:
Thanks for your review. I am not a Python expert, so I do not know what has to be changed in `setup.py`.

Reply:
I had a deeper look into the python build system and I understand now the issue. Fixing it upstream would be the right fix, but the fix on derivation side is not ugly. I added a comment, which explains the issue for cross compilation.